lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -562,8 +563,28 @@ sub prepare_sdaf_project {
     assert_script_run("cd $deployment_dir");
     assert_script_run('mkdir -p ' . log_dir());
 
+    # Calculate SDAF version used for deployment and picks latest -1
+    # SDAF_GIT_AUTOMATION_BRANCH variable will override calculated value
+    my $branch = get_var('SDAF_GIT_AUTOMATION_BRANCH', '');
+    if (!$branch || $branch eq 'latest') {
+        my $tags = script_output("curl -s https://github.com/Azure/sap-automation/tags | grep /releases/tag/ | grep data-view-component | cut -d '/' -f6 | cut -d '\"' -f1");
```

Comment:
I suggest to take github REST  API to get tags, like `curl -s https://api.github.com/repos/Azure/sap-automation/tags`, it returns json data, and more stable than HTML page.

Reply:
Thanks `curl -s https://api.github.com/repos/Azure/sap-automation/tags` shows some tags which are not listed/released in https://github.com/Azure/sap-automation/tags. Do you know why?

Reply:
Thanks, good idea.
Updated code and VR is good: https://openqaworker15.qa.suse.cz/tests/304844#step/configure_deployer/175